apps/force/src/styles/index.scss

```diff
@@ -0,0 +1,223 @@
+.app-page,
```

Comment:
I notice a lot of styling, it looks great and maybe nice way to keep it isolated. However just putting in a comment to ask if we would change again if we move full to tailwind.